lms/static/my.js

```diff
@@ -106,14 +106,39 @@ function trackReadAllNotificationsButton(button) {
   });
 }
 
+function getPostUploadMessage() {
+  const myDropzone = Dropzone.forElement('#demo-upload');
+  const feedbacks = document.getElementById('upload-feedbacks');
+  const matchesSpan = document.getElementById('upload-matches');
+  const missesSpan = document.getElementById('upload-misses');
+  myDropzone.on('success', function() {
+    const args = Array.from(arguments).slice(1)[0];
+    const matches = args['exercise_matches'];
+    const misses = args['exercise_misses'];
+    if (matches.length) {
+      matchesSpan.innerHTML += matches + ',';
+    }
+    if (misses.length) {
+      missesSpan.innerHTML += misses + ',';
+    }
+    feedbacks.classList.remove('d-none');
+  });
+}
+
 window.escapeUnicode = escapeUnicode;
 
 window.addEventListener('load', () => {
-  const codeElement = document.getElementById('code-view').dataset;
-  const solutionId = codeElement.id;
   updateNotificationsBadge();
   trackReadAllNotificationsButton(document.getElementById('read-notifications'));
-  trackCopyButton(document.getElementById('copy-button'), document.getElementById('user-code').textContent);
-  trackShareSolution(solutionId, document.getElementById('share-action'));
-  trackDisableShareButton(solutionId, document.getElementById('cancel-share'));
+  if (document.getElementById('code-view') !== null) {
```

Comment:
Search in the DOM tree is an expansive task - try to minize it by putting #code-view on a const (instead of retrieving in twice)